src/sources/soundsourceffmpeg.h

```diff
@@ -42,7 +42,7 @@ struct ffmpegCacheObject {
 
 class SoundSourceFFmpeg : public SoundSource {
   public:
-    explicit SoundSourceFFmpeg(QUrl url);
+    explicit SoundSourceFFmpeg(const QUrl& url);
     ~SoundSourceFFmpeg();
```

Comment:
Unrelated to your PR -- I was about to suggest adding override to ~SoundSourceFFmpeg() but I realized AudioSource doesn't have a virtual destructor. That seems like a problem since we delete pointers to AudioSource via QSharedPointer<AudioSource> in CachingReaderWorker and AnalyzerQueue. 

As I understand it, this is undefined behavior in C++, right? We have no guarantee ~SoundSourceFFmpeg() will run by the C++ standard.